frontend/src/components/headers/DocumentHeader.tsx

```diff
@@ -130,7 +87,9 @@ function DocumentHeader() {
 						<DownloadMenu />
 					</Stack>
 					<Stack direction="row" alignItems="center" gap={1}>
-						<UserPresence presenceList={presenceList} />
+						<Suspense fallback={<CircularProgress size={24} />}>
+							<UserPresenceList presenceList={presenceList} />
+						</Suspense>
```

Comment:
I have a question.
Why we should use `Suspense` and lazy loading here?

Reply:
The coderabbit suggested considering lazy loading for the UserPresenceList. However, I'm unsure if the UserPresenceList component is large enough to warrant using lazy loading.